docker-entrypoint.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -e
+
+# 1. Check if .env exists; if not, create it from template
+if [ ! -f /app/saskatoon/.env ]; then
+    echo "Creating .env from template"
+    cp /app/saskatoon/env.template /app/saskatoon/.env
```

Comment:
@luciancic I don't find it very clear what is being used from the `.env` file.  BTW how are you managing it? I think one should be generated instead of copying `SASKATOON_SECRET_KEY='<KEY>'`

Reply:
Ah, quite right. I was copying it in case a dev comes in fresh without having manually copied `env.template` into a `.env` file. But I see now that if I set the secret key in docker-compose.yml then there's no need for any of the other values from the env template.